s3/pkg/service/bucket.go

```diff
@@ -30,6 +30,17 @@ import (
 var ENC_KEY_LEN int = 32
 var ENC_IV_LEN int = 16
 
+var CrudSupportedClouds = []string{"aws-s3", "azure-blob", "gcp-s3", "hw-obs"}
+
+func contains(slice []string, item string) bool {
```

Comment:
function name can be optimized as ```isCRUCSupportedCloud``` as contain function is used for the purpose only